test/dynamo/test_dynamic_shapes_variations.py

```diff
@@ -0,0 +1,95 @@
+# Owner(s): ["module: dynamo"]
+from torch._dynamo import config
+from torch._dynamo.testing import make_test_cls_with_patches
```

Comment:
did you really have to rename the file lol. now I can't tell what the diffs are

Reply:
No, I don't have to, but I thought it was more sane like this. I can undo.